:truck::package: Deprecate repository documentation in favor of docs.carto.com (#395)
CartoDB/carto-selfhosted-helm: README.md

# CARTO self-hosted for Kubernetes

![header](./img/header-k8s.png)

> This repository contains the Helm chart for CARTO self-hosted used to deploy CARTO in Kubernetes environments.

Reviewer: ```suggestion
This repository contains the Helm chart to deploy CARTO Self-hosted in an orchestrated container deployment with Kubernetes.
```

> For a Docker flavor of CARTO, please refer to <https://github.com/CartoDB/carto-selfhosted>

Reviewer: ```suggestion
For a Single Virtual Machine deployment (with docker-compose), please refer to <https://github.com/CartoDB/carto-selfhosted>
```
